test/EmitDeadStrippable/EndToEnd.swift

```diff
@@ -0,0 +1,171 @@
+// This version of lld no longer supports MachO, and EDS hasn't been implemented in LLD for ELF or COFF yet, so disable the test, but leave it as a demonstration
+// UNSUPPORTED: target={{.*}}
```

Comment:
This test requires LLD for MachO support, which is disabled in Apple's LLVM fork, so I don't expect this test to pass, just including it for demonstration purposes for now.